.github/workflows/apk.yaml

```diff
@@ -46,12 +46,8 @@ jobs:
     - name: Copy jar files
       run: |
         docker cp $(docker create --rm isaac/isaac:latest-msgs-jar-ubuntu16.04):/src/msgs/devel/share/maven/ .
-        ls -l isaac/apks/isaac_gs_ros_bridge/app/libs/
         rm isaac/apks/isaac_gs_ros_bridge/app/libs/*msgs*
-        ls -l isaac/apks/isaac_gs_ros_bridge/app/libs/
-        find maven -name *.jar
         find maven -name *.jar | xargs cp -t isaac/apks/isaac_gs_ros_bridge/app/libs
```

Comment:
When using find and xargs together in an unattended script, it's a good habit to use `find -print0` and `xargs -0` or switch to `find -exec ... +`, in order to avoid corner cases with filenames that contain unusual characters like spaces.

https://serverfault.com/a/994742